website/docs/r/stream_analytics_output_mssql.html.markdown

```diff
@@ -73,6 +73,12 @@ The following arguments are supported:
 
 * `table` - (Required) Table in the database that the output points to. Changing this forces a new resource to be created.
 
+* `max_batch_count` - (Optional) The max batch count to write to the SQL Database. Defaults to `10000`. Possible values are between `1` and `1073741824`.
+
+* `max_writer_count` - (Optional) The max writer count for the SQL Database. Defaults to `1`. Possible values are `0` and `1`.
+
+-> **NOTE:** `0` means it's based on query partition. `1` means single writer.
```

Comment:
In this particular case I would consolidate this useful info into the description of the field.
```suggestion
* `max_writer_count` - (Optional) The max writer count for the SQL Database. Defaults to `1`. Possible values are `0` which bases the writer count on the query partition and `1` which corresponds to a single writer.

Reply:
Updated